core/blockchain.go

```diff
@@ -478,41 +478,41 @@ func (bc *BlockChain) insert(block *types.Block) {
 	}
 }
 
-// Accessors
+// Genesis Accessors
```

Comment:
This was supposed to be a more general code "section" comment. Bad idea tbh. Might as well convert it into a proper doc for this single method. Please change to:

`Genesis retrieves the chain's genesis block.`